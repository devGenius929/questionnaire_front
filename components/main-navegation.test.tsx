import React from 'react';
import renderer from 'react-test-renderer';
import Navegation from './main-navegation';

describe('Menu', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Navegation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
