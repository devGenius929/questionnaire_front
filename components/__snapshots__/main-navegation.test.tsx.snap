// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<ul
  className="menuprincipalsist dropmenu"
  id="nav-one"
>
  <li
    className=""
    id="ip"
  >
    <a
      href="/informacao-pessoal"
    >
      Informações pessoais
    </a>
  </li>
  <li
    className=""
    id="declaracao"
  >
    <a
      href="/declaracoes"
    >
      Declaração
    </a>
  </li>
  <li
    className=""
    id="resumo"
  >
    <a
      href="/resumo"
    >
      Resumo e verificação
    </a>
  </li>
  <li
    className=""
    id="rankingsist"
  >
    <a
      href="/resultado/ranking"
    >
      Resultado / ranking
    </a>
  </li>
  <li
    className="acessorapido"
  >
    <a
      href="/acesso-rapido"
    >
      Acesso rápido
    </a>
  </li>
</ul>
`;
